src/main/java/it/gov/pagopa/ranking/dto/RankingPageDTO.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.ranking.dto;
+
+import it.gov.pagopa.ranking.model.RankingStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class RankingPageDTO {
+
+    private List<RankingRequestsApiDTO> content;
+    private int pageNumber;
+    private int pageSize;
+    private long totalElements;
+    private long totalPages;
+    private RankingStatus rankingStatus;
+    private LocalDateTime rankingPublishedTimeStamp;
+    private LocalDateTime rankingGeneratedTimeStamp;
+    @Builder.Default
+    private long totalEligibleOk = 0;
+    @Builder.Default
+    private long totalEligibleKo = 0;
```

Comment:
let's rename these, or let's consider the possibility to have a counter separated for the ONBOARDING_KO
As a requirement communicated yesterday, also the ONBOARDING_KO will be stored inside the ranking
@StefanoTerragni could you please check in such way this change could influence this model?

Reply:
Seems to be an impact also on our DB (as a simialr copy to what we have for the section Adherents/Users) from what the lawyers woud like to see, even if we are duplicating same objects. About the case, we should differentiate ELIGIBLE_KO with ONBOARDING_KO (citizen not evaluated by admissibility component) in order to add a new specific variable for it.

Reply:
@StefanoTerragni I suggest to add another counter, totalOnboardingKo.
What do you think about it?

Reply:
@antonioT90 following the convention we used for Eligibile, I agree with you and we go for it.